components/scripts/lib/build-scan-offline.sh

```diff
@@ -34,9 +34,9 @@ find_build_scan_dump() {
 }
 
 read_build_scan_dumps() {
-  local build_scan_csv
+  local build_scan_data
   echo -n "Extracting build scan data"
-  build_scan_csv="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" extract "0,${build_scan_dumps[0]}"  "1,${build_scan_dumps[1]}")"
-  parse_build_scan_csv "$build_scan_csv" "build_cache_metrics_only"
+  build_scan_data="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" extract "0,${build_scan_dumps[0]}"  "1,${build_scan_dumps[1]}")"
+  parse_build_scan_data "build_cache_metrics_only" "$build_scan_data"
```

Comment:
We should drop the quotes around `build_cache_metrics_only`.  We don't quote this elsewhere.

Reply:
I recognize the inconsistency but I intentionally did this. Without the quotes it is indistinguishable from a variable or function call. IMO it would be nice to add quotes in the other places.

Reply:
> it would be nice to add quotes in the other places.

That sounds reasonable to me. As long as how we call the function is consistent, I don't have a strong opinion here.

Reply:
I have addressed this in #388